src/lib/env.js

```diff
@@ -85,11 +87,11 @@ export class Env {
     };
 
     const langToFetch = lang.src === 'pseudo' ?
-      { code: this.defaultLang, src: 'app' } :
+      { code: 'en-US', src: 'app' } :
```

Comment:
I'd like to stop passing the default language to the `Env` constructor because in the future, we should just take the last language in the `langs` array.  This is why I don't mind hardcoding this here for the time being.